SPECS/python-pytest-forked/python-pytest-forked.spec

```diff
@@ -6,13 +6,13 @@ C++ libraries that might crash the process. To use the plugin, simply use the
 --forked argument when invoking py.test.}
 Summary:        py.test plugin for running tests in isolated forked subprocesses
 Name:           python-%{pypi_name}
-Version:        1.4.0
-Release:        2%{?dist}
+Version:        1.6.0
```

Comment:
We are moving to `python 3.12`  (#7730) and it seems this may not be compatible with it as per github.
Do we still want to upgrade ?

Reply:
Thank you for pointing this out @mfrw `python 3.11` is supported by pytest-forked and I didn't find any breaking changes in `3.12` which should cause trouble. 

But I guess we can hold this PR until python upgrade happens.